### DISCUSSION TITLE
vulkan-1.dll 找不到

### DISCUSSION THREAD

**Discussion by biossguo** [BoomingTech/Piccolo#293] [General]
win10-x64         
vs2019

![5DA47B0C-E672-494C-BF9B-2442CCFBBE7A](https://user-images.githubusercontent.com/1680588/177911201-323ce64c-5210-4abf-9374-dbc97615d263.png)

我下载了一个 vulkan-1.dll,放到了 system32 依然不行

**Comment by Ol6rin** [comment]
首先建议安装/重装VulkanSDK试一下，如果不行把dll放到exe相同路径下再试试。下载的dll还需要注意32/64位的问题。

**Reply by biossguo** [reply]
重装了，还是不行，是不是因为我是用的虚拟机的原因呢，mac 笔记本上装了一个虚拟机，虚拟机里面安装的 win10 x64 + vs2019

**Reply by Ol6rin** [reply]
之前也有报告说虚拟机上运行存在问题，我们暂时没有计划支持虚拟机环境。

**Reply by biossguo** [reply]
好的，非常感谢
